### PR TITLE
Updates Discord Invite Link

### PR DESCRIPTION
**Pull request by zafar-hussain**
[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### Describe your change:

- [ ] Add an algorithm?
- [x] Fix a bug or typo in an existing algorithm?
- [x] Documentation change?

### Checklist:

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [ ] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [ ] All new JavaScript files are placed inside an existing directory.
- [ ] All filenames should use the UpperCamelCase (PascalCase) style. There should be no spaces in filenames.
      **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
- [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
- [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

A discord link in Readme.md was updated to the new link to discord invites, namely "https://the-algorithms.com/discord/"

### CODE REVIEW

**Comment by zafar-hussain** [comment]
@reviewers

please, if you can add "hacktoberfest-accepted" label, if this PR is fulfilled

cheers

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ JavaScript Repository of TheAlgorithms, which implements various algorithms and
   [![Checks][checks]][actions]
   [![Contributions Welcome][welcome]](CONTRIBUTING.md)
   [![standard.js][standard-logo]][standard-js]
-  [![Discord chat][chat]][discord-server]
+  [![ chat][chat]][-server]
```
Comment (appgurueu) [inline README.md:14]: Regression

**Comment by appgurueu** [comment]
> @reviewers
> 
> please, if you can add "hacktoberfest-accepted" label, if this PR is fulfilled
> 
> cheers

I'm afraid this PR is probably too trivial to be accepted (I'm also wary of using the TheAlgorithms link rather than a direct link to the Discord).

**Review by zafar-hussain — COMMENTED** [review]
Should I change the link  back to the original, or leave it as it is?

Sorry for a newbie question by me

cheers

**Comment by appgurueu** [comment]
> Should I change the link back to the original, or leave it as it is?

Changing it back would render this PR detrimental. I'm trying to determine whether we should close the PR. @raklaptudirm

@Panquesito7 would probably want the link to be updated.

File: README.md (hunk 1 above)
Comment (Panquesito7) [inline README.md:14]: Please change this back to how it was.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (zafar-hussain) [reply README.md]: Done. Please check and advice.


cheers

**Review by appgurueu — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🎉

**Pull request merged**